python/core/auto_generated/processing/qgsprocessingparameters.sip.in

```diff
@@ -2392,6 +2395,20 @@ Returns the name of the parent layer parameter, or an empty string if this is no
 Sets the name of the parent layer parameter. Use an empty string if this is not required.
 
 .. seealso:: :py:func:`parentLayerParameterName`
+%End
+
+    bool useInlineExpressionEditor() const;
```

Comment:
This would be better suited to inclusion in the freeform parameter metadata instead -- we do that elsewhere for settings which provide very specific changes to the GUI operation of a parameter. Eg. the number of decimal places setting for QgsProcessingParameterNumber

Reply:
Good to know, implemented.